src/environments/environment.prod.ts

```diff
@@ -1,5 +1,6 @@
 export const environment = {
   production: true,
+  appVersion: require('../../package.json').version,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/personalmoney/app-supabase/pullRequest?prid=7587366)
```suggestion
  appVersion: require("../../package.json").version,
```